### PR TITLE
add get_event_to function

### PR DESCRIPTION
**Pull request by weigang992003**

### CODE REVIEW

File: lib/ecto_state_machine.ex [hunk 1]
```diff
@@ -29,6 +29,20 @@ defmodule EctoStateMachine do
         unquote(events) |> Enum.map(fn(x) -> x[:name] end)
       end
 
+      def unquote(:"get_event_to")(event) do
+        unquote(events)
+        |> Enum.find(
```
Comment (asiniy) [inline lib/ecto_state_machine.ex]: This doesn't work well for me. What could be if there are couple of ways to go from point A to point B?

File: lib/ecto_state_machine.ex [hunk 2]
```diff
@@ -29,6 +29,20 @@ defmodule EctoStateMachine do
         unquote(events) |> Enum.map(fn(x) -> x[:name] end)
       end
 
+      def unquote(:"get_event_to")(event) do
```
Comment (asiniy) [inline lib/ecto_state_machine.ex:32]: 1. This doesn't work for me. What could be if there are couple of ways to go from point A to point B?
I think there should be another name for this function. Like, `get_events_to`.

Comment (asiniy) [reply lib/ecto_state_machine.ex:32]: 2. I dislike current interface. It should show not the direction from `nil` or `default` state, it should accept destination state (or sturct contains this state) and final state.

Result should looks like:

``` elixir
User.get_events_to(:unconfirmed, :confirmed)
User.get_events_to(%User{ state: :unconfirmed }, :confirmed)
```

File: test/ecto_state_machine_test.exs [hunk 3]
```diff
@@ -102,4 +102,10 @@ defmodule EctoStateMachineTest do
   test "#events" do
     assert User.rules_events == [:confirm, :block, :make_admin]
   end
+
+  test "#get_event_to" do
```
Comment (asiniy) [inline test/ecto_state_machine_test.exs:106]: 3. According to the 2, please make test coverage bigger than now. Like:

``` elixir
describe "#get_event_to" do
  test "from :unconfirmed" do
    # a lot of stuff here
  end

  test "from :confirmed" # and all possible states here!
end
```

**Review by asiniy — COMMENTED** [review]
I'd also ask you to cover this feature in the docs, but it's not required. Please make these 3

**Comment by weigang992003** [comment]
This is a very personal function, and i do not think it should be merged.
Thanks.

**Pull request closed without merging**